src/main/kotlin/no/nav/familie/oppdrag/service/AvstemmingService.kt

```diff
@@ -0,0 +1,31 @@
+package no.nav.familie.oppdrag.service
+
+import no.nav.familie.oppdrag.avstemming.AvstemmingSender
+import no.nav.familie.oppdrag.repository.OppdragProtokollRepository
+import no.nav.virksomhet.tjenester.avstemming.meldinger.v1.Avstemmingsdata
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+
+@Service
+class AvstemmingService(
+        @Autowired private val avstemmingSender: AvstemmingSender,
+        @Autowired private val oppdragProtokollRepository: OppdragProtokollRepository) {
+
+    fun utførGrensesnittavstemming(listeAvstemmingMeldinger: List<Avstemmingsdata>) {
+
+        LOG.debug("Utfører grensesnittavstemming for ${listeAvstemmingMeldinger.size} antall meldinger.")
+
+        listeAvstemmingMeldinger.forEach {
+            avstemmingSender.sendGrensesnittAvstemming(it)
```

Comment:
Har du noen feel på hvor stor XML-en blir i praksis? OS forventer nok separate start- og sluttmeldinger, men dersom avstemmingsdataen ikke er over 70 MB totalt så kan vi vel sende det i en enkelt melding? Jeg har sendt PDF-er med bilder over MQ og de har aldri vært i nærheten av 70 MB, så tipper at dette også er godt under grensa. Dataen til konsistensavstemmingen må vi kanskje dele opp, da :)

Reply:
Enig! Dette kan legges i en egen oppgave hvor vi undersøker det faktiske behovet for å sende dette i separate meldinger. :)